README.md

```diff
@@ -254,6 +254,29 @@ try {
 catch (e) {
     console.log('share resource image to time line failed', e);
 }
+
+//Code example to share file use shareToSession API on android
+try {
+    var rootPath = fs.DocumentDirectoryPath;
+    var filePath = rootPath + '/email-signature-262x100.png'; // like
+
+    await fs.downloadFile('http://www.ncloud.hk/email-signature-262x100.png', filePath);
+
+    var result = await WeChat.shareToSession({
+        type: 'file',
+        title: 'image file download from network',
+        description: 'share file to friend',
+        mediaTagName: 'email signature',
+        messageAction: undefined,
+        messageExt: undefined,
+        filePath: "file://" + filePath
```

Comment:
Hi @liuchungui 
The prefix "file://" is required for android but redundant for iOS. So, In my new commit #43 , the filePath field will be handled separately, on Android platform, the prefix "file://" will be append in native java code.

So, we don't have to add the prefix in javascript code.